docs/topics/jvm/jvm-spring-boot-add-data-class.md

```diff
@@ -75,7 +76,7 @@ It requires changing the `MessageController` class to respond with a JSON docume
     <deflist collapsible="true">
        <def title="Collections – listOf()">
           <p>The Kotlin Standard Library provides implementations for basic collection types: sets, lists, and maps.<br/>
-          A pair of interfaces represents each collection type:</p>
+          Each collection type can be <i>read-only</i> or <i>mutable</i>:</p>
           <list>
               <li>A <i>read-only</i> interface that provides operations for accessing collection elements.</li>
               <li>A <i>mutable</i> interface that extends the corresponding read-only interface with write operations: adding, removing, and updating its elements.</li>
```

Comment:
This part still mentions interfaces so maybe as an alternative:
* A _read-only_ collection comes with operations for accessing collection elements.
* A _mutable_ collection comes also with write operations for adding, removing, and updating its elements.

WDYT?